packages/sources/elven/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@chainlink/elven-adapter",
+  "version": "0.0.0",
+  "description": "Chainlink elven adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "elven"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "tsc -b",
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "server:dist": "node -e 'require(\"./dist/index.js\").server()'",
+    "start": "yarn server:dist"
+  },
+  "dependencies": {
+    "@chainlink/external-adapter-framework": "0.30.1",
```

Comment:
The checks cover this but looks like there's a new version to bump to `0.30.2`

Reply:
I think that's alright. Just saw the version check isn't required to get this merged so we should be good.

Reply:
Just saw the dependency bump PR has now been merged - will bump Elven to match